rubygem-concurrent-ruby-edge/rubygem-concurrent-ruby-edge.spec

```diff
@@ -0,0 +1,70 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name concurrent-ruby-edge
+%global rubyabi 1.9.1
+
+Summary: Edge concepts for the modern concurrency tools for Ruby
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.0.pre3
+Release: 1%{?dist}
+Group: Development/Languages
+
+License: MIT
+URL: https://github.com/ruby-concurrency/concurrent-ruby
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix}ruby(abi) = %{rubyabi}
+Requires: %{?scl_prefix}ruby(rubygems)
+Requires: %{?scl_prefix}ruby
+BuildRequires: %{?scl_prefix}ruby(abi) = %{rubyabi}
```

Comment:
The build failure (on Fedora) is coming from these lines, you need to put in a check for Fedora and use `ruby(release)` (no specific version) and only use `ruby(abi)` on SCL.  Copy the style that's in other spec files, it's in almost all of them.  Be sure to update both BR and Requires.